cmd/cmds_add.go

```diff
@@ -15,13 +15,14 @@ var cmdsAddCmd = &cobra.Command{
 	Short: "Add a command to run on a specific event",
 	Long:  `Add a command to run on a specific event.`,
 	Args:  cobra.MinimumNArgs(2),
-	Run: python(func(_ *cobra.Command, args []string, d pythonData) {
+	RunE: python(func(_ *cobra.Command, args []string, d *pythonData) error {
 		s, err := d.store.Settings.Get()
 		checkErr(err)
```

Comment:
Since you're changing to `RunE`, what about return errors instead of using the `checkErr` function?